analyzers/tests/SonarAnalyzer.Test/Rules/ConditionalStructureSameImplementationTest.cs

```diff
@@ -37,7 +38,7 @@ public class ConditionalStructureSameImplementationTest
 
         [TestMethod]
         public void ConditionalStructureSameImplementation_Switch_CSharp() =>
-            builderCS.AddPaths("ConditionalStructureSameImplementation_Switch.cs").Verify();
+            builderCS.AddPaths("ConditionalStructureSameImplementation_Switch.cs").WithOptions(ParseOptionsHelper.FromCSharp8).Verify();
```

Comment:
The tests should follow [Proposal B](https://docs.google.com/document/d/1E1aVqiaFvK5bWNtf5fe9ol7VVN2LJaZ3qzvNifEKbAM/edit#heading=h.inqkfjd2vglq) from the test case circle.